packages/trader/src/Services/socket-general.js

```diff
@@ -120,6 +120,10 @@ const BinarySocketGeneral = (() => {
                 common_store.setError(true, { message: response.error.message });
                 break;
             case 'InvalidToken':
+                // if message type equals to cashier, there is no need to logout user
+                if (msg_type === 'cashier') {
```

Comment:
Once branch is updated, this file is removed, please make this change in `packages/core/src/Services/socket-general.js`.

Reply:
I copied the whole invalid token part to the core